source/javascripts/site.js

```diff
@@ -1,27 +1,27 @@
 // This is where it all goes :)
 
-cards = [
- {
- 	prompt: "How do we become more productive?"
- },
- {
- 	prompt: "Name 3 morning activities."
- },
- {
- 	prompt: "Name a favorite restaurant."
- },
-]
-
 var cardIndex = 0;
 
+
+
 window.onload = function() {
+	var topicH1 = document.getElementById("card-topic-header");
+
 	var refreshButton = document.getElementById("refresh-button");
 	var cardPrompt = document.getElementById("card-prompt");
 
-	refreshButton.onclick = function() {
+	function drawCard() {
 		cardIndex = (cardIndex + 1) % cards.length;
 		cardPrompt.innerHTML = cards[cardIndex].prompt;
 	}
-}
 
+	refreshButton.onclick = drawCard;
 
+	//add listeners to the topic headers to change the card heading text
+	document.querySelectorAll(".topic__link").forEach(function(node) {
+		node.addEventListener('click', function(){
+			topicH1.innerText = node.innerText;
+			drawCard();
+		})
+	});
```

Comment:
Maybe add a call to `drawCard()` here so a card is drawn immediately when the page loads